consumer.py

```diff
@@ -56,27 +56,26 @@
 from confluent_kafka import Consumer
 from confluent_kafka import TopicPartition
 
-
 # We run all the modules so that their metrics and handlers get registered
-run_module("thoth.investigator.advise_justification.investigate_advise_justification")
-run_module("thoth.investigator.adviser_trigger.investigate_adviser_trigger")
-run_module("thoth.investigator.build_analysis_trigger.investigate_build_analysis_trigger")
-run_module("thoth.investigator.cve_provided.investigate_cve_provided")
-run_module("thoth.investigator.hash_mismatch.investigate_hash_mismatch")
-run_module("thoth.investigator.inspection_completed.investigate_inspection_completed")
-run_module("thoth.investigator.kebechet_run_url_trigger.investigate_kebechet_run_url_trigger")
-run_module("thoth.investigator.kebechet_trigger.investigate_kebechet_trigger")
-run_module("thoth.investigator.missing_package.investigate_missing_package")
-run_module("thoth.investigator.missing_version.investigate_missing_version")
-run_module("thoth.investigator.package_extract_trigger.investigate_package_extract_trigger")
-run_module("thoth.investigator.package_released.investigate_package_released")
-run_module("thoth.investigator.provenance_checker_trigger.investigate_provenance_checker_trigger")
-run_module("thoth.investigator.qebhwt_trigger.investigate_qebhwt_trigger")
-run_module("thoth.investigator.si_unanalyzed_package.investigate_si_unanalyzed_package")
-run_module("thoth.investigator.solved_package.investigate_solved_package")
-run_module("thoth.investigator.unresolved_package.investigate_unresolved_package")
-run_module("thoth.investigator.unrevsolved_package.investigate_unrevsolved_package")
-run_module("thoth.investigator.update_provide_source_distro.investigate_update_provide_source_distro")
+run_module("thoth.investigator.advise_justification.__init__")
```

Comment:
all lgtm,
just unsure of this.
maybe we should just do a run of this in test and see.

Reply:
sure, will /deploy do the trick. I'm not sure exactly how that works since there are two deployments dependent on this repository.

Reply:
what are the two deployments' dependencies? and yes that should get the image built for investigator and get it deployed in the test cluster.

Reply:
Hey sorry. It's `investigator` and `investigator_metrics`

Reply:
@harshad16 The above does run all the modules. I tested locally by just adding a print statement to one of the modules

Reply:
excellent, thanks @KPostOffice